examples/breathe/breathe.c

```diff
@@ -0,0 +1,34 @@
+#include "ch32fun.h"
```

Comment:
Please add a comment here explaining what this is doing/why.

Reply:
Added.